ui/components/S3Form.tsx

```diff
@@ -130,75 +113,6 @@ const S3ConfigForm = ({ setter }: S3Props) => {
             />
           );
       })}
-
-      <Label
```

Comment:
This PR makes this optional, why is it removing it from UI?

Reply:
Yup good point. Restored the UI for metadata DB which is now toggled via a switch (Screenshot in PR)